src/screens/Home/index.test.tsx

```diff
@@ -1,13 +1,35 @@
 import ActiveRideView from './';
-import renderer, {ReactTestRenderer} from 'react-test-renderer';
+import renderer, {
+  ReactTestRenderer,
+  ReactTestInstance,
+} from 'react-test-renderer';
 import {updateRealTimeRecord, getOrCreateRealtimeRecord} from '../../utils';
+import App from '../../../App';
 import RealtimeDataModel from '../../database/model/realtimeDataModel';
+import {navigate} from '../../lib/navigation';
 
-// mocking call because we aren't rendering
-// under a navigation context.
-jest.mock('@react-navigation/elements', () => ({
-  useHeaderHeight: jest.fn().mockImplementation(() => 200),
-}));
+let tree!: ReactTestRenderer;
+let screen!: ReactTestInstance;
+
+beforeEach(async () => {
```

Comment:
Updated how we do screen tests. Now we load the app and navigate to the screen. It makes sure we always have them wrapped correctly.